alpakaConfig.cmake

```diff
@@ -488,8 +490,13 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
                     LIST(APPEND CUDA_NVCC_FLAGS "-Wno-deprecated-gpu-targets")
                 ENDIF()
 
-                LIST(APPEND CUDA_NVCC_FLAGS "--expt-extended-lambda")
-                LIST(APPEND CUDA_NVCC_FLAGS "--expt-relaxed-constexpr")
+
+                IF(ALPAKA_CUDA_NVCC_LAMBDAS)
+                    LIST(APPEND CUDA_NVCC_FLAGS "--expt-extended-lambda")
```

Comment:
Could you use a name that is more closely related to the flag? `ALPAKA_CUDA_NVCC_EXTENDED_LAMBDA`?

Reply:
But 'expt' is an ugly abbreviation for 'experimental' to begin with.

Reply:
But it is not our flag but nvcc's. So we should not change the naming.

Reply:
okay